code/modules/speech/say_channels/mentormouse.dm

```diff
@@ -0,0 +1,4 @@
+/datum/say_channel/mentor_mouse
+	suppress_speech_bubble = TRUE
+	suppress_say_sound = TRUE
+	channel_id = SAY_CHANNEL_MENTOR_MOUSE
```

Comment:
`affected_by_modifiers` should be set to `FALSE` here, otherwise accents will be applied to mentor chat messages.

Reply:
Added